src/sortStudents.ts

```diff
@@ -1,16 +1,63 @@
-
 export interface Student {
   // describe Student interface
+  name: string,
+  surname: string,
+  age: number,
+  married: boolean,
+  grades: number[],
 }
 
 export enum SortType {
   // describe SortType enum
+  Name = 'name',
+  Surname = 'surname',
+  Age = 'age',
+  Married = 'married',
+  AverageGrade = 'grades',
 }
 
 // create SortOrder type
-export type SortOrder;
+export type SortOrder = 'asc' | 'desc';
 
+export function getAverageGrade(grades: number[]): number {
+  return (grades.reduce((sum: number, n: number) => sum + n, 0)
+  / grades.length);
+}
 
-export function sortStudents(students, sortBy, order) {
+export function sortStudents(
+  students: Student[],
+  sortBy: SortType,
+  order: SortOrder,
+): Student[] {
   // write your function
+
+  const StudentsCopy = [...students];
```

Comment:
```suggestion
  const studentsCopy = [...students];
```
Better to use camelCase for variable names